src/headline_grabber/pipelines/__init__.py

```diff
@@ -14,6 +15,7 @@
         ScrapeText(),
         ScoreSentiment(),
         ClassifySubject(),
+        FilterMaxResults(),
```

Comment:
lets move both ClassifySubject and FilterMaxResults immediately after ScrapeText, so that we don't needlessly use NLP resources for headlines we immediately drop afterward

Reply:
I switched then around to have ClassifySubject follow ScrapeText and FilterMaxResults follows ClassifySubject.